addons/advanced_fatigue/functions/fnc_handleStaminaBar.sqf

```diff
@@ -13,6 +13,11 @@ params ["_stamina"];
 
 private _staminaBarContainer = uiNamespace getVariable [QGVAR(staminaBarContainer), controlNull];
 
+if ((vehicle ACE_player) != ACE_player) exitWith { // Hide when mounted
```

Comment:
I think we're still going to need the if statement here or it will be overwriten